src/main/interface/msp.c

```diff
@@ -1018,7 +1018,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
     case MSP_COMP_GPS:
         sbufWriteU16(dst, GPS_distanceToHome);
         sbufWriteU16(dst, GPS_directionToHome);
-        sbufWriteU8(dst, GPS_update & 1);
+        sbufWriteU8(dst, GPS_update & (GPS_MSP_UPDATE | GPS_DIRECT_TICK));
```

Comment:
With GPS_MSP, toggling bit 0 should be enough (and backward compatible).

Reply:
OK, will revert